pandas/tests/test_graphics.py

```diff
@@ -2953,6 +2953,62 @@ def test_line_colors(self):
         tm.close()
 
     @slow
+    def test_line_colors_and_styles_subplots(self):
+        # GH 9894
+        from matplotlib import cm
+        default_colors = self.plt.rcParams.get('axes.color_cycle')
+
+        df = DataFrame(randn(5, 5))
+
+        axes = df.plot(subplots=True)
+        for ax, c in zip(axes, list(default_colors)):
+            self._check_colors(ax.get_lines(), linecolors=c)
+        tm.close()
+
+        # single color char
+        axes = df.plot(subplots=True, color='k')
+        for ax in axes:
+            self._check_colors(ax.get_lines(), linecolors=['k'])
+        tm.close()
+
+        # single color str
+        axes = df.plot(subplots=True, color='green')
+        for ax in axes:
+            self._check_colors(ax.get_lines(), linecolors=['green'])
+        tm.close()
+
+        custom_colors = 'rgcby'
+        axes = df.plot(color=custom_colors, subplots=True)
```

Comment:
can you also add a second version of this test but with `list(custom_colors)`? (or is that already somewhere else?)